doc/source/_static/js/whatsnew_links.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+	const links = document.getElementsByTagName('a');
```

Comment:
Nit: consistency of quotation marks.  You use double-quotes almost everywhere else.

Reply:
Amended!